parser/SqlParser.ypp

```diff
@@ -1332,6 +1369,83 @@ opt_limit_clause:
     }
   }
 
+opt_window_clause:
+  {
+    $$ = nullptr;
+  }
+  | window_declaration_list
+
+window_declaration_list:
+  window_declaration {
+    $$ = new quickstep::PtrList<quickstep::ParseWindow>();
+    $$->push_back($1);
+  }
+  | window_declaration_list window_declaration {
+    $$ = $1;
+    $$->push_back($2);
+  }
+
+window_declaration:
```

Comment:
I think currently `SqlParser.hpp` is mostly organized in this way:
```
A:
  TOKEN_X SubA1 SubA2
  ...
  ;

SubA1:
  TOKEN_Y SubSubA1 SubSubA2
  ...
  ;

SubSubA1:
  ...
  ;

...

SubA2:
  ...
  ;
```
I.e. kind of infix-order traversal of the grammar tree.

Let's just have this PR in this structure. We can have a standalone PR if there is a proposal for better organization of the grammar rules.